.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -610,7 +610,7 @@ jobs:
           OSSRH_PASSWORD: ${{ secrets.svcs-ossrh-password }}
         with:
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: "releaseEvmMavenCentral --scan -PpublishSigningEnabled=true --no-configuration-cache --no-parallel"
+          arguments: "releaseEvmMavenCentral --scan -PpublishSigningEnabled=true --no-configuration-cache"
```

Comment:
`--no-parallel` no longer needed as we now properly communicate to nexus that all publications "belong together" by using the nexus plugin.